lib/svg2png.js

```diff
@@ -21,21 +19,26 @@ module.exports = async (source, options) => {
 };
 
 async function convertBuffer(source, options) {
-    const browser = await puppeteer.launch();
+    const tempFile = await createTempFile();
+    let browser;
     let output;
 
     try {
-        const page = await browser.newPage();
         const svg = source.toString("utf8");
         const start = svg.indexOf("<svg");
 
-        let html = HTML_PREFIX;
+        const baseUri = options.url ? options.url : "http://localhost/";
```

Comment:
url, not uri

Reply:
I'll get this fixed.

I was actually wondering if the API would be simpler if we simply had a single `base` option. It might be good if this was exposed as a CLI option as I can imagine it'd be useful for converting files downloaded online. But that's a story for another PR, I guess.